### PR TITLE
chore: export reference utils

### PR DESCRIPTION
**Pull request by cezarsmpio**
I will need those utils to fix the embedded entry card.

### CODE REVIEW

**Review by djagya — APPROVED** [review]

File: packages/reference/src/index.tsx [hunk 1]
```diff
@@ -21,3 +21,5 @@ export type {
   MissingEntityCardProps,
   RenderCustomMissingEntityCard,
 } from './common/customCardTypes';
+export { isValidImage } from './utils/isValidImage';
+export { fromFieldValidations } from './utils/fromFieldValidations';
```
Comment (z0al) [inline packages/reference/src/index.tsx:25]: A similar function is also exported from `field-editor-shared` did you try it?

Comment (cezarsmpio) [reply packages/reference/src/index.tsx:25]: Let me check! :)

Comment (cezarsmpio) [reply packages/reference/src/index.tsx:25]: Hum, actually maybe we should move `isValidImage` to `field-editor-shared`?

Comment (z0al) [reply packages/reference/src/index.tsx:25]: It's only used in `isValidImage` but also doesn't seem tied to the reference logic. I'm fine with that.

The two implementations of `fromFieldValidations` seem to be slightly different. Not sure which one is more correct/complete.

Comment (cezarsmpio) [reply packages/reference/src/index.tsx:25]: I think I don't need `fromFieldValidations`, right now I'm only using `isValidImage`.

Comment (z0al) [reply packages/reference/src/index.tsx:25]: Let's not export it then

**Pull request closed without merging**